packages/lisk-api-client/package.json

```diff
@@ -58,7 +58,8 @@
 	},
 	"dependencies": {
 		"@liskhq/lisk-constants": "1.1.1",
-		"axios": "0.18.0"
+		"axios": "0.18.0",
+		"babel-runtime": "6.26.0"
```

Comment:
@shuse2 I am not that familiar with `lerna`, however, I read somewhere that if the dependency is common across all packages we can add it root, so why are we not considering that here?

Reply:
It is only for `devDependencies` which doesn't use the ./bin folder, so it doesn't apply here